app/assets/stylesheets/themes/default/_participant.scss

```diff
@@ -0,0 +1,146 @@
+.participant-content {
+  background: white;
+  padding: 20px;
+  margin-bottom: 20px;
+
+  &__title {
+    color: $base-color;
+    font-size: $default-font-size;
+    margin: 0;
+  }
+
+  &__description {
+    margin: 1em 0 0 0;
+  }
+}
+
+.participant-titles {
+  border-top: 2px solid rgba($base-color, 0.2);
+  margin-top: 20px;
+  padding-top: 10px;
+  position: relative;
+
+  li {
+    margin-bottom: 10px;
+    margin-left: 1.5em;
+
+    &:before {
+      content: ' ';
+      border: transparent solid 0.3em;
+      border-left-color: $base-color;
+      position: absolute;
+      height: 0;
+      width: 0;
+      left: 0.5em;
+      margin: 0px;
+      margin-top: 0.4em;
+      padding: 0px;
+    }
+
+    strong {
+      font-size: $default-font-size - 2;
+      padding: 2px 10px;
+      color: white;
+      background-color: $base-color;
+      border-radius: $input-border-radius;
+    }
+  }
+
+  @include media($smaller-tablet) {
+    li {
+      font-size: $default-font-size - 2;
+    }
+  }
+}
+
+@include media($larger-tablet) {
```

Comment:
Can you embed media query into class name for consistency?
``` scss
.participant-content {
  @include media($larger-tablet) {
    // ...
  }
}
```

See also [`_post.scss`](https://github.com/bm-sms/daimon_news-layout/blob/c93b45154a49f79752b64147a668151f877a79f5/app/assets/stylesheets/themes/default/_post.scss#L11-L13).